index.html

```diff
@@ -9,19 +9,48 @@
 </head>
 <body>
 
-  <h1 class="title"> All Awesome Books</h1>
+  <header>
+    <nav class="nav-bar">
+      <h2 class="nav-title">Awesome Books</h2>
+      <ul class="links-list">
+        <li class="nav-link active">List</li>
+        <li class="nav-link middle-link">Add New</li>
+        <li class="nav-link">Contact</li>
+      </ul>
+    </nav>
```

Comment:
- [ ] Good job by adding the navigation :clap: But  I noticed that your link are not clickables,  according to the project requirement, when you click on any of the links, the main area changes to display only one of the three content sections. In your case, all page content is displayed. Please fix it to make your project work as expected . :+1: 
![image](https://user-images.githubusercontent.com/6615394/206569492-6ac269cb-e8eb-4317-89e1-d8b28c07b4fa.png)

#### Your app
![Screenshot from 2022-12-08 21-11-50](https://user-images.githubusercontent.com/6615394/206568386-bea9816a-dd5c-46b6-b908-bdb3f9841ce4.png)

Reply:
@codecaiine  CAN YOU CHAEK NOW